packages/template-retail-react-app/app/components/drawer-menu/index.jsx

```diff
@@ -97,6 +99,14 @@ const DrawerMenu = ({isOpen, onClose = noop, onLogoClick = noop, root}) => {
     const supportedLocaleIds = l10n?.supportedLocales.map((locale) => locale.id)
     const showLocaleSelector = supportedLocaleIds?.length > 1
 
+    useEffect(() => {
+        var spinner = document.getElementById('spinner')
+        var catNav = document.getElementById('category-nav')
```

Comment:
First of all, let's use `const` instead of `var`.

Second, the react-y way for accessing DOM elements is by using [refs](https://reactjs.org/docs/hooks-reference.html#useref).

```jsx
function DrawMenu() {
  const spinner = useRef(null);
  
  return (
      <div ref={spinner} />
  );
}
```

Lastly, do we really need to access the spinner elements? it looks like the spinners are conditionally rendered based on the state `showLoading`. Could it be as simple as 

```jsx
<div
  id="category-nav"
  aria-live="polite"
  aria-busy={showLoading}
  aria-atomic="true"
>
```